src/nidcpower/metadata/functions_addon.py

```diff
@@ -154,4 +150,96 @@
             'description': 'Returns the number of channels.',
         },
     },
+    # Public function that wraps driver function but returns datetime object instead of individual items
+    'GetLastExtCalLastDateAndTime': {
+        'codegen_method': 'public',
+        'returns': 'ViStatus',
+        'python_name': 'get_ext_cal_last_date_and_time',
+        'real_datetime_call': 'GetExtCalLastDateAndTime',
```

Comment:
who is this `real_datetime_call` used?

Reply:
It is used in datetime_wrappers.py,mako to know what the actual function name to call is.

Reply:
But doesn't this wrapper call the private python method `_get_ext_cal_last_date_and_time()` anyway?

Reply:
I wanted one template that would work for all drivers. Of course they are not consistent. Not in names and not in parameter order.